README.md

```diff
@@ -41,3 +41,203 @@ Package can then be placed directly under `./charts/<centos|redhat>/<chart-name>
 
 You can also use Helm CLI commands, please refere to [Using Helm Guide](https://helm.sh/docs/intro/using_helm/) for detailed instructions on how to use the Helm client.
+
+## How to contribute to this repository
+
+In case of Helm chart is related to CentOS, CentOS Stream 8, or CentOS Stream 9 then use [centos](charts/centos) directory.
+
+In case of Helm chart is related to RHEL system then use [redhat](charts/redhat) directory.
+
+As soon as your Helm chart follows directory structure as mentioned above, let's test it.
+
+### General notes for Helm charts
+Each helm chart name has to contain file OWNERS. E.g. [OWNERS](./charts/redhat/postgresql-imagestreams/OWNERS)
+
+Each helm chart change, in [templates](./charts/redhat/postgresql-persistent/0.0.1/templates) or [OWNERS](./charts/redhat/postgresql-persistent/OWNERS) file
+has to bump version
+
+### How to test Helm chart
+
+As a prerequisite to testing Helm charts, you need to be connected to the OpenShift 4 cluster using the oc binary generated from the cluster itself.
+To get access to OpenShift 4 cluster please ask phracek@redhat.com.
+
+#### Create a Helm chart package
+
+Before any testing proposes and deploying Helm chart to OpenShift cluster, you have to create a package using command:
+
+```commandline
+$ helm package <path_to_helm_chart>
+```
+
+E.g. Using postgresql-persistent Helm chart:
+
+```commandline
+$ helm package charts/redhat/postgresql-persistent/0.0.1
+Successfully packaged chart and saved it to: <FULL_PATH_TO_CWD>/helm-charts/postgresql-persistent-0.0.1.tgz
+```
+
+#### Deploy Helm chart to OpenShift
+
+You can deploy the Helm chart to the OpenShift cluster using command:
+
+```commandline
+$ helm install <helm_chart_name> <tar_ball_path>
+```
+
+E.g Using postgresql-persistent Helm chart:
+
+```commandline
+$ helm install postgresql-imagestreams postgresql-imagestreams-0.0.1.tgz
+NAME: postgresql-imagestreams001
+LAST DEPLOYED: Tue Apr  4 10:29:33 2023
+NAMESPACE: pgsql-13
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+To check if Helm chart is really deployed, run command:
+
+```bash
+helm list
+```
+
+Examples of the `helm test` output:
+
+```commandline
+$ helm list
+NAME                      	NAMESPACE	REVISION	UPDATED                              	STATUS  	CHART                        	APP VERSION
+postgresql-imagestreams  	pgsql-13 	1       	2023-04-04 10:29:33.878674 +0200 CEST	deployed	postgresql-imagestreams-0.0.1	0.0.1
+postgresql-persistent     	pgsql-13 	1       	2023-03-28 16:59:56.498783 +0200 CEST	deployed	postgresql-persistent-v0.0.1
+```
+
+### How to verify Helm chart
+
+As soon as a Helm chart is ready and tested let's verify it. The first we have to create a package by command:
```

Comment:
Does this really create a package? Manual pages says `The test command runs the tests for a release.`

Reply:
I have rephrased it.